src/compiler/abstract_compiler.nit

```diff
@@ -491,6 +516,7 @@ endif
 		end
 
 		# Compile each required extern body into a specific .o
+		var java_files = new Array[ExternFile]
```

Comment:
th move of java_files seems unrelated with the commit

Reply:
Maybe, but it was obviously at the wrong place when I added the documentation where it was previously.